UltraStar Play/Assets/Common/Audio/Recording/CamdAudioSamplesAnalyzer.cs

```diff
@@ -0,0 +1,171 @@
+using System;
+using System.Collections.Generic;
+using UniRx;
+using UnityEngine;
+
+public class CamdAudioSamplesAnalyzer : IAudioSamplesAnalyzer
+{
+    /** There are 49 halftones in the hearable audio spectrum (C2 to C6 (1046.5023 Hz)). */
```

Comment:
This statement confuses me.
A normal piano keyboard has 88 keys and each give a different sound (and I can hear that sound).
Thus, I expect there are more that 49 halftones, right?

Reply:
Maybe there are 49 halftones a human can sing, i.e., 49 halftones can be created using a human voice?

Reply:
ah yes, 49 halftones of range (C2 - C6) for typical human singing according to https://en.wikipedia.org/wiki/Vocal_range :

- Soprano: the highest female voice, being able to sing C4 (middle C) to C6 (high C), and possibly higher.
- Mezzo-soprano: a female voice between A3 (A below middle C) and A5 (2nd A above middle C).
- Contralto: the lowest female voice, F3 (F below middle C) to E5. Rare contraltos possess a range similar to the tenor.
- Tenor: the highest male voice, B2 (2nd B below middle C) to A4 (A above Middle C), and possibly higher.
- Baritone: a male voice, G2 (two Gs below middle C) to F4 (F above middle C).
- Bass: the lowest male voice, E2 (two Es below middle C) to E4 (the E above middle C).

Will fix the code comment in the next commit.